.github/workflows/hello_world/helloworld.ts

```diff
@@ -0,0 +1,4 @@
+import { Resonate } from "@resonatehq/sdk/dist/index.js";
```

Comment:
can we have this heelowrodl app cover more?

meaning let's import context, register a async function that prints out hello world and resonate.run() it with sqlite in memory .

Reply:
resonate.schedule(), resonate.start() as well we should have minimum confidence in all of our existing features in the build process before it publishes to our users.

Reply:
Agreed! I think the hello world function can be a resonate function that returns a string. Let me see if I can merge my changes today and then we could write two resonate functions (one for generator and one for async) conforming to the new changes.

Reply:
also it would be nice to test the import `@resonatehq/sdk/dist/index.js` (like you are above) before pushing and `then` try the npm install and try with the regular import to verify the published artifact (not just the code) is ok.

Reply:
Let's merge this PR and come back to the hello world test program later. The sdk is changing too fast, it would just add to merge conflicts for now. I think better to hold off for a week or two.